src/components/block_details/types.ts

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {
+  Data,
+  TableHead,
+} from './components/signatureDialog/types';
```

Comment:
This needs to get moved up to this file and imported in to signatureDialog instead.
Parent -> Children ✓
Children -> Parent ╳
We are following a unidirectional flow here

also rename signatureDialog -> signature_dialogue